cmd/zoekt-sourcegraph-indexserver/merge.go

```diff
@@ -47,12 +67,16 @@ func doMerge(dir string, targetSizeBytes, maxSizeBytes int64, simulate bool) err
 	for ix, comp := range compounds {
 		debug.Printf("compound %d: merging %d shards with total size %.2f MiB\n", ix, len(comp.shards), float64(comp.size)/(1024*1024))
 		if !simulate {
+			start := time.Now()
 			stdOut, stdErr, err := callMerge(comp.shards)
+			metricShardMergingDuration.Observe(time.Since(start).Seconds())
```

Comment:
Not blocking on this, but ideally we'd want to have a _separate_ metric for the duration of failed merging operations

Reply:
We don't really expect that many failed merges. It should be a rare event. According to our log, there were 0 errors during merging over last 30 days. I think the error counter already gives us a good enough signal for now.

Reply:
Even if we don't have failures so far (great!), I think we should standardize on always having a normalized `error` label in latency histograms.

Reply:
done